operators/src/source/ogr_source/mod.rs

```diff
@@ -470,6 +535,74 @@ impl FeaturesProvider<'_> {
         Ok(())
     }
 
+    fn set_attribute_filters(&mut self, attribute_filters: &[AttributeFilter]) -> Result<()> {
+        for filter in attribute_filters {
+            let attribute = escape_identifier(&filter.attribute);
+            for range in &filter.ranges {
+                let sql = match &range {
+                    StringOrNumberRange::String(s) => {
+                        format!(
+                            "{attribute} >= {start} AND {attribute} <= {stop}",
+                            attribute = attribute,
+                            start = escape_literal(s.start()),
+                            stop = escape_literal(s.end())
+                        )
+                    }
+                    StringOrNumberRange::Float(n) => format!(
+                        "{attribute} >= {start} AND {attribute} <= {stop}",
+                        attribute = attribute,
+                        start = n.start(),
+                        stop = n.end()
+                    ),
+                    StringOrNumberRange::Int(n) => format!(
+                        "{attribute} >= {start} AND {attribute} <= {stop}",
+                        attribute = attribute,
+                        start = n.start(),
+                        stop = n.end()
+                    ),
+                };
```

Comment:
I like this abstraction. Could we optimize `=` if `start == stop`?

Reply:
62908f874ce3f398f188593bac5bf66d3fc6bd46